libs/components/forms/src/lib/modules/file-attachment/file-drop/file-drop.component.ts

```diff
@@ -71,7 +80,7 @@ const MIN_FILE_SIZE_DEFAULT = 0;
     SkyThemeModule,
   ],
 })
-export class SkyFileDropComponent implements OnDestroy {
+export class SkyFileDropComponent implements OnDestroy, ControlValueAccessor {
```

Comment:
Are we handling a "disabled" state?

Reply:
i dont see anything in the styles or template that would support a disabled state, but i can ask design 👁️

Reply:
confirmed with design that file drop does not have a disabled state